tests/endpoints/mocked.py

```diff
@@ -0,0 +1,16 @@
+class TestMockedEndpoints:
+    def test_mocked_response(self, mocked_router, client):
+        result = client.get('/users')
+
+        assert result.status_code == 200
+        assert result.json() == mocked_router.routes[0].responses[0].body
+
+        result = client.get('/non-existent-path')
```

Comment:
This feels like it should be separate test case

Reply:
Yes, but 'it does mean changing ~~the bulb'~~ changing lot's of other tests too for consistency, since this is the approach used in most of them.

I'll look into it.